websocket.go

```diff
@@ -182,6 +181,7 @@ func (t wsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}()
 
 	// Loop for each message
+outerReadLoop:
 	for err == nil {
 		mt, buf, rerr := ws.ReadMessage()
```

Comment:
I thought there was some Read deadline here. Perhaps this is a follow-on work item...